pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -204,11 +194,6 @@ func (service *SecurityPolicyService) buildTargetTags(obj *v1alpha1.SecurityPoli
 	rule *v1alpha1.SecurityPolicyRule, ruleIdx int, createdFor string,
 ) []model.Tag {
 	basicTags := service.buildBasicTags(obj, createdFor)
-	sort.Slice(*targets, func(i, j int) bool {
```

Comment:
Remove this sort function for building target tags, the reason is that this function will change rule target content. the generated rule hash in buildRuleID is different from the one generated before sort. 
we will generate NSX rule ID during expanding rules, by this time, there is no rule sort. 
After sort, the rule hash will be changed when it builds rule hash for target group ID.

So, we need to keep rule hash consistency, we need to remove this rule target sort function.